powersimdata/design/investment/investment_costs.py

```diff
@@ -155,21 +170,36 @@ def get_transformer_mult(x, bus_reg, ac_reg_mult, xfmr_lookup_alerted=set()):
 
     bus_reg.drop(["lat", "lon"], axis=1, inplace=True)
 
+    # Add extra information to branch data frame
+    branch = grid_new.branch
+    branch.loc[:, "kV"] = branch.apply(
+        lambda x: bus.loc[x.from_bus_id, "baseKV"], axis=1
+    )
```

Comment:
Should this be equivalent to `branch["kV"] = bus.loc[branch["from_bus_id"], "baseKV"]`?

Reply:
Done (simplified this one and the two after it).